interval/tools.go

```diff
@@ -2,9 +2,29 @@ package interval
 
 import (
 	"fmt"
+	"github.com/vertgenlab/gonomics/numbers"
 )
 
 // CoordsToString takes in a interval and returns a string in the format of chr:start-end
 func CoordsToString(i Interval) string {
 	return fmt.Sprintf("%s:%v-%v", i.GetChrom(), i.GetChromStart(), i.GetChromEnd())
 }
+
+// IntervalSize calculates the size of the interval
+func IntervalSize(i Interval) int {
+	return i.GetChromEnd() - i.GetChromStart()
+}
+
+// OverlapSize calculates the size of the overlap betwen 2 intervals, assuming they are from the same genome
+func OverlapSize(a, b Interval) int {
+	if a.GetChrom() != b.GetChrom() {
```

Comment:
It might be nice anyways to have a Overlap() function in the interval package that just returns a bool if to intervals overlap or not. Then this function could look a little cleaner. Something like
`if !Overlap(a, b) {
return 0
} else {
return numbers.Min(a.GetChromEnd(), b.GetChromEnd()) - numbers.Max(a.GetChromStart(), b.GetChromStart())
}`

Purely stylistic though because this code works as is. Theres a similar set of functions in Bed if you want to look at them

Reply:
I want to add this so will have more commits!